internal/session/repository_session.go

```diff
@@ -170,8 +170,7 @@ func (r *Repository) ListSessions(ctx context.Context, opt ...Option) ([]*Sessio
 	var args []interface{}
 
 	inClauseCnt := 0
-	switch {
-	case len(opts.withScopeIds) != 0:
+	if len(opts.withScopeIds) != 0 {
```

Comment:
It might be worth checking with @jimlambrt to see if he had a reason for this to be either/or instead of and -- I think what you changed it to will work, but not sure offhand why it was like that in the first place.

Reply:
(BTW I fully realize this _must_ be changed for this to work, but it seems like it should have been this way to begin with, hence the question)

Reply:
Hum... looks as if I believed you'd want to search for user sessions across scopes or just list sessions by scope.  Honestly, don't remember why I went down this path.